mcs/class/monodoc/Monodoc/providers/EcmaDoc.cs

```diff
@@ -578,16 +602,19 @@ internal static string MakeOperatorSignature (XElement member, out string member
 
 		static XElement ExtractClassSummary (XDocument typeDoc)
 		{
-			string name = typeDoc.Root.Attribute("Name").Value;
-			string fullName = typeDoc.Root.Attribute("FullName").Value;
-			string assemblyName = typeDoc.Root.Element("AssemblyInfo") != null ? typeDoc.Root.Element("AssemblyInfo").Element("AssemblyName").Value : string.Empty;
-			var docs = typeDoc.Root.Element("Docs");
-			var summary = docs.Element("summary") ?? new XElement("summary");
-			var remarks = docs.Element("remarks") ?? new XElement("remarks");
+			string name = typeDoc.Root.Attribute ("Name").Value;
```

Comment:
:/ I know, sorry ... it just seemed such an egregious break from the coding standards that had just made it through to master. Is it verboten to do whitespace changes ever? Would it be better to do have a "whitespace only" pull request to fix things like this? or better to just leave it as is until that code gets rewritten entirely at some point?